samples/bluetooth/central_hr/README.rst

```diff
@@ -1,12 +1,13 @@
-.. _bluetooth_central_hr:
+.. zephyr:code-sample:: ble_central_hr
+   :name: Heart-rate Monitor (Central)
+   :relevant-api: bluetooth
 
-Bluetooth: Central / Heart-rate Monitor
-#######################################
+   Connect to a BLE heart-rate monitor and read heart-rate measurements.
 
 Overview
 ********
 
-Similar to the :ref:`Central <bluetooth_central>` sample, except that this
+Similar to the :zephyr:code-sample:`Central <bluetooth_central>` sample, except that this
```

Comment:
overriding the text also works for the `:code-sample:` tag?

Reply:
checked the render, seems that it does, cool

Reply:
Yep! 🪄

Reply:
I'm done with the first review round btw, thanks for the thankless grunt work :+1: Didn't have the stamina to push it through myself.

Reply:
Thanks! I didn't expect such a thorough review for a first pass tbh :) but this gives me the feedback that I was originally looking for which is that you guys would be welcoming the change :)